package.json

```diff
@@ -1,14 +1,13 @@
 {
   "name": "purecloud-streaming-client",
-  "version": "11.3.1",
+  "version": "12.0.0",
   "description": "client for the PureCloud Streaming APIs (websocket/xmpp interface)",
   "repository": "https:github.com/purecloudlabs/purecloud-streaming-client",
   "license": "MIT",
   "publishConfig": {
     "registry": "https://registry.npmjs.org/"
   },
-  "main": "dist/streaming-client.min.cjs",
-  "browser": "dist/streaming-client.js",
+  "main": "dist/src/client.js",
```

Comment:
I decided to just point at the es6 module and let the consuming application transpile as it needs.